src/platforms/dart/migration.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Migration Guide
+sidebar_order: 1000
+redirect_from:
+  - /platforms/dart/migrate/
```

Comment:
```suggestion
```

not necessary as we're just creating this page

Reply:
Agreed - we only create redirects when we're moving the location of exising content. As this is new content, there's no need to add a redirect.